### PR TITLE
Issue 2 view entries

### PR DESCRIPTION
**Pull request by hellohien**
Screen Shot 2021-03-31 at 10 49 09 AM
Screen Shot 2021-03-31 at 10 49 35 AM

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -23,5 +27,41 @@ function addEntry(event) {
   $form.reset();
```
Comment (codingcodymiller) [inline js/main.js:28]: You should be adding the newly submitted entry to the list of entries here, too. You are currently only making DOM trees for new elements when the page loads, but you also need to generate one for each new entry and add it to the DOM.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by hellohien** [comment]
Screen Shot 2021-04-01 at 11 09 31 AM
Screen Shot 2021-04-01 at 11 09 35 AM

File: js/main.js [hunk 2]
```diff
@@ -5,23 +5,73 @@ var $previewImage = document.querySelector('.preview-image');
...
+  return $list;
+}
+
+for (var i = 0; i < data.entries.length; i++) {
+  $entriesList.appendChild(entryList(data.entries[i]));
+}
+
+function changeView(view) {
+  for (var index = 0; index < $containers.length; index++) {
+    var dataView = event.target.getAttribute('data-view');
```
Comment (codingcodymiller) [inline js/main.js]: You are still using `event` in here, even though you are not using this as an event handler in your submit function. You should create a function that has a `view` parameter that is meant to be the name of the view to switch to as a string, so you can call that function and pass in the view name from `data-view` as an argument.

File: css/styles.css [hunk 3]
```diff
@@ -2,27 +2,44 @@
   font-family: 'Proza Libre', sans-serif;
 }
 
+body {
+  background-color: rgb(245, 245, 245);
+}
+
+.hidden {
+  display: none;
+}
```
Comment (codingcodymiller) [inline css/styles.css]: There is already a rule for the `hidden` class in `layout.css`, this can be removed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js [hunk 4]
```diff
@@ -27,7 +27,7 @@ function addEntry(event) {
   $previewImage.src = 'images/placeholder-image-square.jpg';
   $form.reset();
   $entriesList.prepend(entryList(entries));
-  changeView();
+  getDataValue('entries');
```
Comment (codingcodymiller) [inline js/main.js]: This should just be `changeView("entries")`

Comment (codingcodymiller) [reply js/main.js]: `getDataValue` is a event handler function, and there is no event here.

File: js/main.js [hunk 5]
```diff
@@ -61,17 +61,21 @@ for (var i = 0; i < data.entries.length; i++) {
...
-    if ($containers[index].dataset.view === dataView) {
+    if ($containers[index].dataset.view === view) {
       $containers[index].className = 'container hidden';
     } else {
       $containers[index].className = 'container';
     }
   }
 }
 
+function getDataValue(view) {
```
Comment (codingcodymiller) [inline js/main.js]: No `view` parameter needed here.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by hellohien** [comment]
Screen Shot 2021-04-01 at 3 19 47 PM
Screen Shot 2021-04-01 at 3 19 54 PM

**Comment by hellohien** [comment]
Screen Shot 2021-04-01 at 5 47 24 PM
Screen Shot 2021-04-01 at 5 47 20 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**